src/main/java/joules/task/Event.java

```diff
@@ -0,0 +1,60 @@
+package joules.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import joules.Store;
+
+/**
+ * Represents an event task with a start and end date.
+ * An {@code Event} extends {@link Task} and includes {@link LocalDate}
+ * fields for the start and end dates. It can be stored and displayed
+ * with formatted dates.
+ */
+public class Event extends Task {
+    /** Start date of the event */
+    protected LocalDate from;
+
+    /** End date of the event */
+    protected LocalDate to;
```

Comment:
Similar to Deadline.java Line 16, variable names can be lengthened with added context.
```suggestion
    /** Start date of the event */
    protected LocalDate startDate;

    /** End date of the event */
    protected LocalDate endDate;
```